.github/workflows/release.yml

```diff
@@ -22,31 +22,66 @@ jobs:
         run: |
           echo "${{ inputs.version_number }}" | grep -P '^v\d+\.\d+\.\d+(-pre[A-Za-z0-9-]*)?$'  
```

Comment:
once the release is finished succesfully, do we need to put the Unrelease header again in the file?

Reply:
from what I see [here](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2134/files#diff-08c41df88f0b622e21d5036ae226546549ef4bcf07b674a920a76d97c3c9a9d1R26), Unreleased header is never removed and the new versions are always put under the unreleased

Reply:
Correct the Unreleased header is preserved at the top of the changelog, [example commit](https://github.com/AgustinBettati/terraform-provider-mongodbatlas/commit/8b1ecfc6cedf83e1d79a3862e997ae7154467583).